bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/api/GoEStatusResponseDTO.java

```diff
@@ -30,7 +30,7 @@
     @SerializedName("ast")
     public Integer accessConfiguration;
 
-    @SerializedName("amp")
+    @SerializedName("amx")
```

Comment:
I would not replace it, as that disables the ability, to set a value, that should still be available after restart. Better would be to add it as a second property and recommend it in the PV charge sample in documentation.